### DISCUSSION TITLE
✅ Copy & paste not working on OSX

### DISCUSSION THREAD

**Discussion by d34vbd5** [isontheline/pro.webssh.net#676] [General]
Hi,
i am unable to copy and paste from OSX to SSH session. Tried it with different servers, not able to paste anything while copy%pasting on the OS itself works fine.

**Comment by isontheline** [comment]
Hi @d34vbd5 

What's your version of WebSSH and macOS?

**Reply by d34vbd5** [reply]
OSX 12.4 (21F79)
WebSSH Version 19.1 (920)

**Comment by d34vbd5** [comment]
I just found out I am able to paste via the Top Edit Menu and selecting paste there. I am still unable to paste via Right Click -> Paste or CMD + V

First Screenshot: Edit -> Paste is possible
Second Screenshot: Right Click -> Paste or CMD + V not possible 
![Bildschirmfoto 2022-07-07 um 11 44 43](https://user-images.githubusercontent.com/17470504/177744652-c269e706-cebc-47f4-ad0d-e7cf4addf523.png)
![Bildschirmfoto 2022-07-07 um 11 44 52](https://user-images.githubusercontent.com/17470504/177744697-a4e45f09-d552-4a12-b053-21241b230000.png)

**Reply by isontheline** [reply]
Right click + paste have never been implemented

Could you upgrade to 19.3?

**Reply by d34vbd5** [reply]
Seems to work again via CMD + V

Thanks

**Reply by isontheline** [reply]
Thanks for your feedback @d34vbd5
